task/build-image-index/0.1/build-image-index.yaml

```diff
@@ -185,7 +185,7 @@ spec:
         add:
           - SETFCAP
 
-  - image: quay.io/konflux-ci/mobster@sha256:c5d05d4f3962153cde582c1e0f0e1d9979e714070e62b04a2213c61974ca2af9
+  - image: quay.io/konflux-ci/mobster@sha256:0e924f70c0998fac59dc513f5b482f0779c1e0457ebb56e1b8294e4f65fb0ecb
```

Comment:
@Allda @jedinym FYI this image isn't getting grouped with the other mobster updates. I think it's because it doesn't have a tag